guides/Getting Started/GettingStarted.md

```diff
@@ -30,6 +30,37 @@ new Client({
 }).login('your-bot-token');
 ```
 
+### Extending the client
```

Comment:
I think this should be removed, it's not something related with Klasa, but related with JavaScript. The rest (client options update) looks fine to me.